dali/base/dafdesc.cpp

```diff
@@ -2099,6 +2154,9 @@ void CFileDescriptor::serializeParts(MemoryBuffer &mb,unsigned *partlist, unsign
     queryProperties().serialize(mb);
     if (sdesc)
         sdesc->serializeSub(mb);
+    IPropertyTree *t = &queryHistory();
+    if (t)
```

Comment:
afaics - this cannot be null by this time, so there should be no misleading check here.